plugins/scaffolder-backend/src/scaffolder/actions/builtin/publish/gitlab.ts

```diff
@@ -63,6 +66,21 @@ export function createPublishGitlabAction(options: {
             type: 'string',
             description: `Sets the default branch on the repository. The default value is 'master'`,
           },
+          defaultCommitMessage: {
+            title: 'Default Commit Message',
```

Comment:
@Rugvip `defaultCommitMessage` was used because we have same property in scaffolder configuration in app-config.yaml [here](https://github.com/backstage/backstage/blob/e0e15f3169df2db6e789589fbac587d3e4cdc611/app-config.yaml#L292), let me know if it still needs to change to `gitCommitMessage`.

Reply:
come to think of it, since the commit message is mutable while creating a template, changing it to `gitCommitMessage ` would make more sense, thanks for the suggestion @Rugvip 🙏🏼